README.md

```diff
@@ -6,15 +6,15 @@ guidelines:
   is a different problem.
   - Since naming issues may arise (e.g., same problem with different number of
     variables), there must be an ID column;
-- The collection of two or more solver run (`DataFrame`s), is a
+- The collection of two or more solver runs (`DataFrame`s), is a
   `Dict{Symbol,DataFrame}`, where each key is a solver;
 
 Package objectives:
-- [ ] Print to latex (WIP in Optimize);
-- [ ] Print to pretty markdown table;
-- [ ] Produce performance profiles;
+- [ ] Print to latex (WIP in Optimize).
+- [ ] Print to pretty markdown table.
+- [ ] Produce performance profiles.
```

Comment:
Sorry, I meant that only the last semicolon should be a full stop.

Reply:
Fixed